lib/Mojolicious/Plugin/AssetPack/Manual/Include.pod

```diff
@@ -31,8 +33,8 @@ L<asset|Mojolicious::Plugin::AssetPack/asset> helper:
 
   %= asset "my-asset.css"
 
-This will result in a single html link tag in production mode, and a list of
-link tags in development mode.
+This will result in a list of link tags in development mode and a single html
+link tag in production mode.
```

Comment:
Awesome.

Maybe it should also say ```C<link>``` as to indicate that "link" is code? I think the same goes other places in this document.